app/account/services_views.py

```diff
@@ -0,0 +1,24 @@
+import typing as tp
+
+
+def create_new_account(data: tp.Dict[str, tp.Union[str, bool, int]]) -> tp.Dict[str, tp.Union[str, bool, int]]:
```

Comment:
you can upgrade to python 3.10, then these type hints would be way more readable
```suggestion
def create_new_account(data: dict[str, str | bool | int) -> dict[str, str | bool | int]:
```

Reply:
yes, I know about it, but can I do it eventually after another functionality?